plugins/mcreator-localization/help/default/item/when_right_clicked_block.md

```diff
@@ -1,5 +1,9 @@
 The procedure is executed when the player right-clicks on a block with this item in his hand.
 
 The procedure should return an action result of type SUCCESS/CONSUME if the item interacted with the block, 
-FAIL if the interaction failed, and PASS if there was no interaction. 
-If the procedure doesn't return any value, the result action type will be PASS.
\ No newline at end of file
+FAIL if the interaction failed, and PASS if there was no interaction.
+

```

Comment:
The translation for this is:

Cette procédure est exécutée quand un joueur fait un clic droit sur un bloc avec cet item dans sa main.
Cette procédure devrait retourner une résultante d'action de type SUCCÈS/CONSUMMÉ si l'item a interagi avec le bloc, FAIL si l'interaction n'a pas marché, et PASS si il n'y a pas eu d'interaction.

Si la procédure ne retourne aucune valeur, le type de la résultante d'action sera PASS.

Si vous voulez que la procédure "Lors d'un clic droit" ne soit appelée que quand une entité fait un clic droit dans l'air avec cet item, cette procédure devrait toujours retourner SUCCÈS/CONSUMMÉ